temboardui/plugins/monitoring/static/js/daterangepicker.vue.js

```diff
@@ -230,6 +230,6 @@
 
   function notify() {
     this.$emit('update:from', dateMath.parse(this.editRawFrom));
-    this.$emit('update:to', dateMath.parse(this.editRawTo));
+    this.$emit('update:to', dateMath.parse(this.editRawTo, true));
```

Comment:
What is this new argument? If there is a way to use keyword argument, it would be clearer to me.

Reply:
Keyword arguments don't exist in javascript.
See https://github.com/dalibo/temboard/blob/master/temboardui/plugins/monitoring/static/js/datemath.js#L19 and https://github.com/dalibo/temboard/blob/master/temboardui/plugins/monitoring/static/js/datemath.js#L110

Reply:
Ok, thanks; fine by me then.